src/test/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepTest.java

```diff
@@ -605,7 +610,12 @@ public void invalidChoiceParameterValue() throws Exception {
 
     @Test
     public void buildStepDocs() throws Exception {
-        SnippetizerTester.assertDocGeneration(BuildTriggerStep.class);
+        try {
+            SnippetizerTester.assertDocGeneration(BuildTriggerStep.class);
+        } catch (Exception e) {
+            // TODO: Jenkins 2.236+ broke structs-based databinding and introspection of PasswordParameterValue, JENKINS-62305.
+            assumeThat(e.getMessage(), not(containsString("There's no @DataBoundConstructor on any constructor of class hudson.util.Secret")));
```

Comment:
Assume that it does _not_ contain that phrase? So some unrelated NPE will be silently ignored?!

Reply:
Yes good catch, I meant to always rethrow the exception so that it would only be skipped if the text matches and otherwise the test will fail.

Reply:
Fixed in afc8024.